src/mscorlib/shared/System/Text/StringBuilder.cs

```diff
@@ -954,12 +954,45 @@ public StringBuilder Remove(int startIndex, int length)
             return this;
         }
 
-        public StringBuilder Append(bool value) => Append(value.ToString());
+        public StringBuilder Append(bool value)
+        {
+            if (value.TryFormat(RemainingCurrentChunk, out int charsWritten))
```

Comment:
`bool.ToString` doesn't allocate. Is there a benefit to using `TryFormat` here vs. the previous simpler implementation?

Reply:
Nope, I'll revert that. I was in copy-and-paste mode :)